frontend/pages/hosts/details/cards/About/About.tsx

```diff
@@ -21,6 +23,41 @@ const About = ({
   munki,
   mdm,
 }: IAboutProps): JSX.Element => {
+  const renderPublicIp = () => {
+    if (aboutData.public_ip !== "---") {
```

Comment:
also a good place for `DEFAULT_EMPTY_CELL`

Reply:
@RachelElysia not here?

Reply:
did some swaps that you can use if you like - https://github.com/RachelElysia/fleet/pull/114